src/components/DiagnosticReport/DiagnosticReportTable.js

```diff
@@ -6,61 +6,138 @@ import TableContainer from "@material-ui/core/TableContainer";
 import TableHead from "@material-ui/core/TableHead";
 import TableRow from "@material-ui/core/TableRow";
 import Paper from "@material-ui/core/Paper";
-import CircularProgress from "@material-ui/core/CircularProgress";
 import DiagnosticReportTableStyles from "./DiagnosticReportTable.style";
+import Typography from "@material-ui/core/Typography";
+import { useParams } from "react-router-dom";
 
 import { toIndiaDate } from "../../constants";
-import getNestedObject from "./../../utils/getNestedObject";
 
-const DiagnosticReportTable = ({ data }) => {
-  return data ? (
-    <DiagnosticReportTableStyles>
-      <TableContainer component={Paper}>
-        <Table className="observation-table" aria-label="simple table">
-          <TableHead>
-            <TableRow className="table-head">
-              <TableCell align="left">Report Date</TableCell>
-              <TableCell align="left">Status of the report</TableCell>
-              <TableCell align="left">Report</TableCell>
-              <TableCell align="left">Result</TableCell>
-              <TableCell align="left">Performer</TableCell>
-            </TableRow>
-          </TableHead>
-          {data.map(entry => {
-            return (
-              <TableBody>
-                <TableRow>
-                  <TableCell align="left">
-                    {entry.effectiveDateTime
-                      ? toIndiaDate(entry.effectiveDateTime)
-                      : "-"}
-                  </TableCell>
-                  <TableCell align="left">{entry.status}</TableCell>
-                  <TableCell align="left">
-                    {entry.code ? entry.code.text : ""}
-                  </TableCell>
-                  <TableCell align="left"></TableCell>
-                  <TableCell align="left"></TableCell>
-                </TableRow>
-              </TableBody>
-            );
-          })}
-        </Table>
-      </TableContainer>
-    </DiagnosticReportTableStyles>
+const DiagnosticReportTable = ({ data, consentReqId }) => {
```

Comment:
should we be calling it a table anymore? Why not just DiagnosticReportsComponent?

Reply:
Make sense. I will rename these files

Reply:
I have another PR here as this branch is already merged.
https://github.com/ProjectEKA/hiu-ui/pull/39